docs/usage/bulk_AIRRseq/bulk_tutorial.md

```diff
@@ -0,0 +1,161 @@
+# nf-core/airrflow: Bulk Airr-seq tutorial
+
+This tutorial provides step by step introductions on how to run nf-core/airrflow on bulk AIRR-seq data.
+
+## Pre-requisites
+
+> [!INSTALLATION]
+> If you are new to Nextflow and nf-core, please refer to [this page](https://nf-co.re/docs/usage/installation) on how to set up Nextflow and a container engine needed to run this pipeline. At the moment, nf-core/airrflow does NOT support using conda virtual environments for dependency management, only containers are supported. Make sure to [test your setup](https://nf-co.re/docs/usage/introduction#how-to-run-a-pipeline) before running the workflow on actual data.
+
+For the purpose of running this tutorial on your local machine, we recommend a docker installation.
+
+To install docker, follow the instructions [here](https://docs.docker.com/engine/install/). After installation docker on linux, don't forget to check the [post-installation steps](https://docs.docker.com/engine/install/linux-postinstall/).
+
+## Testing the pipeline with built-in tests
+
+Once you have set up your Nextflow and container (docker or singularity), test the airrflow pipeline with built-in test.
+
+```bash
+nextflow run nf-core/airrflow -r 4.2.0 -profile test,docker --outdir test_results
+```
+
+If the tests run through correctly, you should see this output in your command line:
+
+```bash
+output:
+```
+
+## Running airrflow pipeline
+
+### Datasets
+In this tutorial, we will use bulk BCR sequencing data from two subjects in fastq format. The first subject has 3 samples from lymph node and 1 sample from brain lesion while the second subject has 3 samples from lymph node and 3 samples from brain lesion. You don't need to download the samples bacause the links to the samples are already provided in the samplesheet.
+
+### Preparing files needed
+To run the Airrflow pipeline on bulk BCR/TCR sequencing data, several files must be prepared in advance.
+
+- A tab-seperated samplesheet containing the information of each sample. Details on the requied columns of a samplesheet are available [here](https://nf-co.re/airrflow/usage#input-samplesheet).
+
+- A configuration file requiring memory, cpu and time. Before setting the configuration file, we recommend verifying the available memory and cpus on your system. Otherwise, exceeding the system's capacity may result in unexpected errors.
+
+- Information on bulk library generation method(protocol).
```

Comment:
@Vivian0105 it would be great to also provide the samplesheet and configuration file example for this tutorial for download.